CD2/enrollments.md

```diff
@@ -23,4 +23,41 @@ WHERE en.value.course_id = <COURSE_ID>
     AND en.value.type = 'StudentEnrollment'
 ORDER BY us.value.name DESC
 ;
+```
+
+## Get enrolled courses and course instructors
+
+*This query finds course courses with active enrollment for given students, and also returns instructor name and email list.
```

Comment:
Maybe update the description.  The query isn't really finding students since the StudentEnrollment is not included.  Possible rewrite: This query finds courses with an active enrollment for given Canvas users and also returns the instructor name and email list.

Reply:
updated!